Makefile

```diff
@@ -836,6 +836,16 @@ openstack_update_run:
 	$(eval $(call vars,$@,openstack))
 	bash scripts/openstack-update.sh
 
+.PHONY: update_services
+update_services:
+	$(eval $(call vars,$@,openstack))
+	bash scripts/openstack-update-services.sh
+
+.PHONY: update_system
+update_system:
```

Comment:
Could we get some in-line comments describing these targets, i.e. like L851?

Reply:
Comments added, let me know if those in-lines are enough. There is plan for next sprint to add more comprehensive documentation in https://github.com/openstack-k8s-operators/dev-docs/blob/main/edpm_update_overview.md